tools/reviewer/job/sync/SyncTool.java

```diff
@@ -0,0 +1,60 @@
+/*
+ * Copyright 2018 The StartupOS Authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.google.startupos.tools.reviewer.job.sync;
+
+import com.google.startupos.common.flags.Flags;
+import com.google.startupos.tools.reviewer.localserver.service.Protos;
+
+import java.io.BufferedReader;
+import java.io.IOException;
+import java.io.InputStreamReader;
+
+/*
+ * Usage:
+ * bazel build //tools/reviewer/job/sync:sync_tool
```

Comment:
`bazel run` also builds, so no need for a separate `build` command.

Reply:
Done.